src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -24,7 +25,7 @@ interface Props {
   basePath?: HttpStart['basePath'];
   dataTestSubj?: string;
   onClick?: Function;
-  navigateToUrl: CoreStart['application']['navigateToUrl'];
+  navigateToUrl: ApplicationStart['navigateToUrl'];
```

Comment:
Not directly related to the PR, but I spotted that during the dependency analysis so I fixed it. `chrome` no longer wrongly depends on `CoreStart` that way.